tests/unit/test_state.py

```diff
@@ -376,7 +522,7 @@ def test_verify_onlyif_list_cmd(self):
             "__sls__": "tests.cmd",
             "__env__": "base",
             "__id__": "check onlyif",
-            "onlyif": ["/bin/true", "/bin/false"],
+            "onlyif": ["exit 0", "exit 1"],
```

Comment:
Why did you have to change this test? Shouldn't the true/false test case still work?

Reply:
Hey @Ch3LL, I've fixed also this test in order to avoid the usage of `/bin/true|false` here because it causes issues and modifies the expected test behavior in platforms where `/bin/true|false` are not available, i.a. Windows.